ShopifySharp/Services/Order/OrderCancelOptions.cs

```diff
@@ -11,6 +11,14 @@ public class OrderCancelOptions
         [JsonProperty("amount")]
         public decimal? RefundAmount { get; set; }
 
+        /// <summary>
+        /// The three letter code (ISO 4217) for the currency used for the refund.
+        /// The currency of the refund that's issued when the order is canceled. 
+        /// Required for multi-currency orders whenever the `amount` property is provided.
+        /// </summary>
+        [JsonProperty("currency")]
+        public string RefundCurrency { get; set; }
```

Comment:
@HeldyHeileng for consistency, please always use the same name in c# as in the API (casing excepted).

Reply:
Thank you! Fixed naming